src/graphql/schema.js

```diff
@@ -3,7 +3,8 @@ import {
   GraphQLObjectType,
   GraphQLString,
   GraphQLSchema,
-  GraphQLList
+  GraphQLList,
```

Comment:
You're going to have to make updates to the schema on the front-end to reflect the schema changes.